thunder/core/symbol.py

```diff
@@ -526,9 +527,16 @@ def __eq__(self, other):
         return (self.sym, self._var_args, self._var_output) == (other.sym, other._var_args, other._var_output)
 
     def rhs(self):
-        return BoundSymbolRHS(
-            self.sym, tuple(tree_flatten(self._var_args)[0]), tuple(tree_flatten(self._var_kwargs)[0])
-        )
+        def make_hashable(x: Any) -> tuple | FrozenDict:
```

Comment:
This function looks more like "maybe_make_hashable" as there's a fallback path that just returns the input that may or may not be hashable?

Reply:
I don't know if that is still the case in the latest revision